modules/connatixBidAdapter.js

```diff
@@ -308,6 +333,22 @@ export const spec = {
       params['us_privacy'] = encodeURIComponent(uspConsent);
     }
 
+    window.addEventListener('message', function handler(event) {
+      if (!event.data || event.origin != 'https://cds.connatix.com') {
+        return;
+      }
+
+      if (event.data.type === 'cnx_all_identity_providers_resolved') {
+        this.removeEventListener('message', handler);
+        event.stopImmediatePropagation();
+      }
```

Comment:
_Optional_
`cnx_all_identity_providers_resolved`
should we keep the type simple and more ambiguous like `cnx_all_resolved` ?

Reply:
I like it being specific